css-display-3/display-contents-tr-001-ref.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Reftest Reference</title>
+<link rel="author" title="Rune Lillesveen" href="mailto:rune@opera.com">
+<p>You should see the word PASS below.</p>
+<table cellpadding="0" cellspacing="0">
+    <td>P<span>A</span></td>
+    <td>S<span>S</span></td>
+</table>
```

Comment:
Not quite, the anonymous tr box still gets generated (in the non-reference too), but just one. If display: contents wasn't working, you'd have two rows instead.